layouts/partials/navbar.html

```diff
@@ -13,14 +13,14 @@
     <div class="menu">
         <ul class="menu-items">
             {{ range .Site.Menus.main }}
-                <a href="{{ .URL | relLangURL }}" class="menu-item">{{ i18n .Identifier }}</a>
+                <a href="{{ .URL }}" class="menu-item">{{ i18n .Identifier }}</a> <!--| relLangURL -->
             {{ end }}
         </ul>
     </div>
-    {{ $relativePath := replace (replace .Permalink .Site.BaseURL "") (printf "%s/" .Site.Language) "" }}
+    <!-- {{ $relativePath := replace (replace .Permalink .Site.BaseURL "") (printf "%s/" .Site.Language) "" }} -->
```

Comment:
I'm not sure this works. Commenting curly braces often produce errors with hugo.